ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2386,8 +2386,8 @@ periodics:
     - args:
       - --config-dir=./ci-operator/config/openshift/assisted-service
       - --current-promotion-namespace=edge-infrastructure
-      - --current-release=ocm-2.7
```

Comment:
we are also going to need ``assisted-service-scripts`` and ``assisted-service-index``, because assisted-installer and other components are dependent on them. e.g.:
https://github.com/openshift/release/blob/cb6c0886f432e55a88e199c04c51e8b4f6be1cd0/ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml#L14-L21

Reply:
Please ignore the comment above. Configurations of this PR are relevant to git repos and not image-streams